apps/explorer/src/ui/header/NetworkSelect.tsx

```diff
@@ -136,7 +138,30 @@ function CustomRPCInput({
     );
 }
 
-function NetworkSelectPanel({ networks, onChange, value }: NetworkSelectProps) {
+function NetworkVersion({
+    label,
+    version,
+}: {
+    label: string;
+    version: number | string;
+}) {
+    return (
+        <div className="flex justify-between bg-gray-40 py-2 px-4">
+            <Text variant="subtitle/normal" color="steel">
+                Sui {label}
+            </Text>
+            <Text variant="subtitle/normal" color="steel">
+                v {version}
```

Comment:
Space looks kinda weird

Reply:
removed the space for now - was torn on how to abbreviate